iop/fs/bdmfs_fatfs/src/ff.c

```diff
@@ -3385,8 +3387,9 @@ static FRESULT mount_volume (	/* FR_OK(0): successful, !=0: an error occurred */
 	if (SS(fs) > FF_MAX_SS || SS(fs) < FF_MIN_SS || (SS(fs) & (SS(fs) - 1))) return FR_DISK_ERR;
 #endif
 
-	/* Find an FAT volume on the drive */
-	fmt = find_volume(fs, LD2PT(vol));
+	/* Check for a valid file system on the device. */
+	fmt = check_fs(fs, 0);
```

Comment:
What does this do?

Reply:
I think this check is for checking if after all mountings we have FatFS compatible system. This actually may be good idea, cause I once corrupted NTFS drive with wLaunchELF that incorrectly handled it it as FAT32 filesystem in partition table.

Reply:
Great! Thank you for clarifying and fixing this bug.